cmd/packer-sdc/internal/fix/gocty_test.go

```diff
@@ -101,7 +101,6 @@ func TestFixGoCty_Unfixed(t *testing.T) {
 		},
 		{
 			name:       "out of date fix",
-			versionStr: "1.13.1",
```

Comment:
Since that's the last test that looks for the version, we can remove this attribute from the test structure.
On a side-note, what did that test check for, is it still relevant?